### PR TITLE
Adding support to the simulator embed for ignoring the aspect ratio

### PR DESCRIPTION
**Pull request by riknoll**
Our current simulator embed forces the aspect ratio of the iframe. This just adds a query parameter to the URL for ignoring the aspect ratio and filling the iframe. Also fixes an unrelated bug where the embedded sim page ignored the reset message.

### CODE REVIEW

**Comment by pelikhan** [comment]
Could you test these changes with microbit?

File: webapp/public/run.html [hunk 1]
```diff
@@ -22,7 +22,6 @@
             position: relative;
             background: transparent;
             width: 100%;
-            padding-bottom: 81.96%;
```
Comment (pelikhan) [inline webapp/public/run.html]: does this get injected later on? or is it useless?

**Comment by riknoll** [comment]
Works fine on mbit!

File: webapp/public/run.html (hunk 1 above)
Comment (riknoll) [reply webapp/public/run.html]: It gets set on the element. No idea why it was also specified in the page because it always gets ignored.

**Pull request merged**